src/wagtail_ai/static_src/custom.d.ts

```diff
@@ -1,6 +1,11 @@
 /* eslint-disable no-unused-vars */
 export {};
 
+export interface WagtailAiConfiguration {
+  wagtailAiPrompts: Array<Prompt>;
```

Comment:
You can just use the `Prompt[]` syntax.

Reply:
Resoved in def70f2e2fa386e2fcaf08371da29c0e83b0e82e.